pin-unpin-panel@anaximeno/files/pin-unpin-panel@anaximeno/5.4/applet.js

```diff
@@ -45,6 +45,9 @@ class PinUnpinPanelApplet extends Applet.IconApplet {
 		this.signalsManager = new SignalManager.SignalManager(null);
         this.signalsManager.connect(global.settings, "changed::" + Panel.PANEL_AUTOHIDE_KEY, this.on_panels_autohide_state_changed, this);
 
+		this.default_pin_icon_path = `${metadata.path}/../icons/pin-symbolic.svg`;
+		this.default_unpin_icon_path = `${metadata.path}/../icons/unpin-symbolic.svg`;
```

Comment:
I think you can use relative paths here if you want.

Reply:
Uhm, I see, thanks for pointing that.